blazor/datagrid/column-chooser.md

```diff
@@ -9,117 +9,436 @@ documentation: ug
+            {
+                int code = 10;
+                for (int i = 1; i < 2; i++)
+                {
+                    Orders.Add(new OrderData(10248,  "France", 33.33,new DateTime(1996,07,07), "Reims"));
+                    Orders.Add(new OrderData(10249,  "Germany", 89.76, new DateTime(1996, 07, 12), "Münster"));
+                    Orders.Add(new OrderData(10250,  "Brazil",78.67, new DateTime(1996, 07, 13), "Rio de Janeiro"));
+                    Orders.Add(new OrderData(10251,  "Belgium", 55.65, new DateTime(1996, 07, 14), "Lyon"));
+                    Orders.Add(new OrderData(10252, "Venezuela",11.09, new DateTime(1996, 07, 15), "Charleroi"));
+                    Orders.Add(new OrderData(10253, "Venezuela",98.98, new DateTime(1996, 07, 16), "Lyon"));
+                    Orders.Add(new OrderData(10254, "Belgium", 78.75, new DateTime(1996, 07, 17), "Rio de Janeiro"));
+                    Orders.Add(new OrderData(10255, "Germany", 44.07, new DateTime(1996, 07, 18), "Münster"));
+                    Orders.Add(new OrderData(10256, "France", 67.74, new DateTime(1996, 07, 19), "Reims"));
+                    code += 5;
+                }
+            }
+            return Orders;
+        }
+        public int? OrderID { get; set; }                
+        public DateTime OrderDate { get; set; }      
+        public string ShipCountry { get; set; }
+        public double Freight { get; set; }
+        public string ShipCity { get; set; }
+    }
+{% endhighlight %}
+{% endtabs %}
 
-    public string[] ToolbarItems = new string[] { "ColumnChooser" };
+{% previewsample "https://blazorplayground.syncfusion.com/embed/BDLgWMNMIHuMlAUe?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+
+## Customize column chooser dialog size
+
+The column chooser dialog in Syncfusion Blazor Grid comes with default size, but you can modify its height and width as per your specific needs using CSS styles.
 
-    public List<EmployeeData> Employees { get; set; }
+To customize the column chooser dialog size, you can use the following CSS styles:
 
+```csharp
+<style> 
+    .e-grid .e-dialog.e-ccdlg {         
+        max-height: 600px !important; 
+        width: 300px !important; 
+    } 
+    .e-grid .e-ccdlg .e-cc-contentdiv { 
+        height: 250px !important; 
+        width: 250px !important;         
+    } 
+</style> 
+```
+> Here, **!important** attribute is used to apply custom styles since the column chooser dialog position will be calculated dynamically based on content.
+
+This can be demonstrated in the following sample:
+
+{% tabs %}
+{% highlight razor tabtitle="Index.razor" %}
+@using Syncfusion.Blazor.Grids
+@using BlazorApp1.Data
+
+<SfGrid ID="Grid" DataSource="@Orders" ShowColumnChooser="true" Toolbar=@ToolbarItems>
+    <GridColumns>
+        <GridColumn Field=@nameof(OrderData.OrderID) HeaderText="Order ID" ShowInColumnChooser="false" TextAlign="TextAlign.Right" Width="120"></GridColumn>
```

Comment:
ShowInColumnChooser property is not required here

Reply:
removed